app/components/windows/OverlayWindow.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
Just curious - Why didn't you use TSX for this component when all others are in TSX?  Was this code that Adrian started?

Reply:
Oh - looks like this component isn't used currently?  Assuming this is a relic from when Adrian was working on interactivity?

Reply:
correct on both accounts